### PR TITLE
Update the Hardhat tutorial to OVM 2.0

### PR DESCRIPTION
**Pull request by qbzzt**
Closing: INT-127

### CODE REVIEW

File: hardhat/README.md [hunk 1]
```diff
@@ -64,17 +69,26 @@ This process downloads and starts an Optimistic Ethereum network of one node.
...
+   git clone https://github.com/ethereum-optimism/optimism.git -b experimental
    ```
+   <!-- TEMO-OVM2.0 -->
 
-2. Start the Optimistic Ethereum node. This process downloads the images
+1. Copy a docker-compose file for OVM 2.0:
+
+   ```sh
+   cd optimism/ops
+   wget https://raw.githubusercontent.com/ethereum-optimism/optimism-tutorial/20211008-ovm20/ovm20-temp/docker-compose-ovm20.yml
```
Comment (K-Ho) [inline hardhat/README.md:80]: Is this really what we want as part of the devex of spinning up getting started project?

File: ovm20-temp/docker-compose-ovm20.yml [hunk 2]
```diff
@@ -0,0 +1,211 @@
+# TODO: Prefix all env vars with service name
```
Comment (K-Ho) [inline ovm20-temp/docker-compose-ovm20.yml:1]: Why does this live here? I'd rather do this the right way and not need to include a docker compose file in our simple getting started tutorial

**Review by K-Ho — CHANGES_REQUESTED** [review]
I'd request we do this right and actually nail down the end-to-end devex for spinning up a local network rather than adding hacks into our official tutorial

File: hardhat/README.md [hunk 3]
```diff
@@ -62,18 +67,20 @@ but they should be similar for other Linux versions and other platforms.
 This process downloads and starts an Optimistic Ethereum network of one node.
 
 1. Clone the [Optimism monorepo](https://github.com/ethereum-optimism/optimism).
+   Not that until we officially release OVM 2.0 you need to clone the `regenesis/0.5.0`
```
Comment (SmitV) [inline hardhat/README.md:70]: _Note_ :)

**Pull request merged**